scripts/apps/vocabularies/views/vocabulary-config-modal.html

```diff
@@ -66,6 +66,14 @@ <h3 class="modal__heading" ng-show="!vocabulary._id"><span translate>Create</spa
                             </label>
                         </div>
                     </li>
+                    <li>
+                        <div>
+                            <label>
+                                <input type="radio" name="field_type" value="picture_video" ng-model="vocabulary.field_type">
```

Comment:
I'm not sure it's a good idea to have such a field type, imo would be better to have field_type=media which has some custom logic like it's a drag & drop with upload on click and use some other field (in options) for supported mimetypes. otherwise it will complicate the logic all over the place

Reply:
it would also imply there could be `video_audio`, `picture_audio`, `picture_video_audio` etc